autogluon/utils/tabular/features/generators/auto_ml_pipeline.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from .pipeline import PipelineFeatureGenerator
+from .category import CategoryFeatureGenerator
+from .datetime import DatetimeFeatureGenerator
+from .identity import IdentityFeatureGenerator
+from .text_ngram import TextNgramFeatureGenerator
+from .text_special import TextSpecialFeatureGenerator
+from ..feature_metadata import R_INT, R_FLOAT
+
+logger = logging.getLogger(__name__)
+
+
+class AutoMLPipelineFeatureGenerator(PipelineFeatureGenerator):
+    """
+    Pipeline feature generator with simplified arguments to handle most Tabular data including text and dates adequately.
+    This is the default feature generation pipeline used by AutoGluon when unspecified.
+    For more customization options, refer to PipelineFeatureGenerator and BulkFeatureGenerator.
+
+    Parameters
+    ----------
+    enable_raw_features : bool, default True
+        Enables raw feature types to be kept.
+        This is typically any feature which is not of the types ['object', 'category', 'datetime'].
+        Appends IdentityFeatureGenerator() to the generator group.
+    enable_categorical_features : bool, default True
+        Enables 'object' and 'category' feature types to be kept and processed into memory optimized category features.
+        Appends CategoryFeatureGenerator() to the generator group.
+    enable_datetime_features : bool, default True
+        Enables 'datetime' features and 'object' features identified as 'datetime_as_object' features to be processed as integers.
+        Appends DatetimeFeatureGenerator() to the generator group.
+    enable_text_special_features : bool, default True
+        Enables 'object' features identified as 'text' features to generate 'text_special' features such as word count, capital letter ratio, and symbol counts.
+        Appends TextSpecialFeatureGenerator() to the generator group.
+    enable_text_ngram_features : bool, default True
+        Enables 'object' features identified as 'text' features to generate 'text_ngram' features.
+        Appends TextNgramFeatureGenerator(vectorizer=vectorizer) to the generator group.
+    vectorizer : CountVectorizer, default None
```

Comment:
why not write out the explicit default CountVectorizer used as it was before? this is more clear for user to see how they can alter things

Reply:
added